### ISSUE TITLE
Segmentation fault with LMMS

### ISSUE THREAD

**Issue by StarbotArc** [LMMS/lmms#7091]
### Bug Summary
Everytime I start messing around with the Calf 5 Band Equalizer and get to low tones, my LMMS just segfaults.

#### Steps to reproduce
I really don't know, but I just suspect it's related to the Calf Equalizers.
I'll just write what idea with Kicker for it to segfault.
1. Make a Kicker instance.
2. Put down Calf Equalizer.
3. Modify it at will, and have one of the Frequences be somewhere low toned.

#### Expected behavior
Low tones generally shouldn't crash LMMS.

#### Actual behavior
It crashes when I equalize and amplify it to lower tones.

#### Screenshot
![image](https://github.com/LMMS/lmms/assets/53960197/9e73ef8f-38d3-4717-b514-de0feb562683)

#### Affected LMMS versions
LMMS 1.3.0-alpha.1.483+g67ce16777

#### Logs
<details>
  <summary>Click to expand</summary>
<pre>

</pre>
</details>

**Comment by michaelgregorius** [comment]
I cannot reproduce this in a Linux build of a rather recent master. @StarbotArc, can you provide a file that will lead to a crash upon playing?

**Comment by Veratil** [comment]
I cannot reproduce on a master branch Windows build as well. (but generally duplicating your settings there made Kicker feedback to itself and gained to earshattering levels)

**Comment by StarbotArc** [comment]
> I cannot reproduce this in a Linux build of a rather recent master. @StarbotArc, can you provide a file that will lead to a crash upon playing?

I've tried, but Github doesn't allow me to post it as it's not a supported type unfortunately.

**Comment by michaelgregorius** [comment]
@StarbotArc, can you please zip the file and upload it here? Github accepts zip files.

**Comment by StarbotArc** [comment]
I would, but I realized now that I've modified the file enough on stable that I might of just loss what makes it crash because I was more focused on music making and didn't realize up until now. I am trying to produce it again, and upgraded back up to get it working, though maybe a bit too far as I am on 1.3.0-alpha.1.518+gdd53bec31 now. I am currently having no luck with getting LMMS to crash.

Another thing I'll add is that I do remember that the master channel did hit peak when I was making the hard-kick even though there was a hard limiter. I might be misremembering it, but maybe an important detail?

If I do get it to crash, I'll try to get a zip out for you all to try.
